src/redux/actions/genes/loadMarkerGenes.js

```diff
@@ -0,0 +1,49 @@
+import {
+  MARKER_GENES_ERROR, MARKER_GENES_LOADING, MARKER_GENES_LOADED,
+} from '../../actionTypes/genes';
+
+import { fetchCachedWork } from '../../../utils/cacheRequest';
+
+const REQUEST_TIMEOUT = 60;
+
+const loadMarkerGenes = (experimentId, resolution) => async (dispatch, getState) => {
+  const { backendStatus } = getState().experimentSettings;
+
+  const body = {
+    name: 'MarkerHeatmap',
+    nGenes: 2,
+    type: 'louvain',
```

Comment:
This `type` value is hardcoded here, but it depends on the clustering settings in the API : https://github.com/biomage-ltd/api/blob/45f57f0dbbfbc50538cf5cebb66e25d0cdccd725/src/utils/hooks/markerGenesWorkRequest.js#L13

will there be any impact?

Reply:
These values are the same as the ones we use for clustering (for marker genes a clustering is run and then the marker genes are calculated from that result).

I'd say that if having this hardcoded should be changed then having hardcoded `type: louvain` in `runCellSetsClustering` should be changed too, so I changed it there too.

Reply:
nGenes doesn't matter, it is hardcoded at 10 in the worker and it ignores this parameter (but still expects it).